cdap-distributions/src/packer/scripts/cookbook-setup.sh

```diff
@@ -28,6 +28,11 @@ done
 ### TODO: remove this hack when chef-cookbooks/ark#181 is solved
 knife cookbook site install ark 2.1.0 || die "Cannot fetch ark cookbook 2.1.0"
 
+# Due to limitations on the Chef version required, pull these manually
+knife cookbook site install build-essential 7.0.3 || die "Cannot fetch build-essential 7.0.3"
```

Comment:
just to confirm, initial cdap install will bring in the latest build-essential, and then this will downgrade?  should be fine, just want to doublecheck